README.md

```diff
@@ -161,10 +161,33 @@ logging config format for content details.
--  Specifying "Datastore Cluster" name during volume creation is not supported. Datastore clusters (as a part of Storage DRS) is a VC feature and not available on individual ESX. [#556](https://github.com/vmware/docker-volume-vsphere/issues/556)
 -  Volume creation using VFAT filesystem is not working currently. [#1327](https://github.com/vmware/docker-volume-vsphere/issues/1327)
--  Volume is not shown when running ```docker volume ls``` and ```vmdkops_admin volume ls``` command after performing Storage vMotion of VM when volume is attached to VM. It is an expected behavior since Storage vMotion changes the location and name of attached vmdk files.
-We only see this when trying to do Storage vMotion with volume attached. So avoid Storage vMotion when volume is attached. [#1618](https://github.com/vmware/docker-volume-vsphere/issues/1618)
+-  Volume of filesystem type ext4dev cannot be attached to a container. ext4dev needs to be mounted as ext4. [#1617](https://github.com/vmware/docker-volume-vsphere/issues/1617)
+
+# Interop with vSphere features
+
+**SvMotion, XvMotion**
+
+- Volume is not shown when running ```docker volume ls``` and ```vmdkops_admin volume ls``` command after performing Storage vMotion or XvMotion of a VM when volume is attached to the VM. It is an expected behavior since Storage vMotion or XvMotion changes the location and name of attached vmdk files.
+We only see this when trying to do Storage vMotion or XvMotion with volume attached. So please avoid Storage vMotion or XvMotion when volume is attached. [#1618](https://github.com/vmware/docker-volume-vsphere/issues/1618)
+
+**Storage DRS**
+
+- Volume creation succeeds on a datastore which is in maintenance mode. Datastore maintenance mode is a VC feature and ESX is not aware if any datastore is put in maintenance mode. [#1651](https://github.com/vmware/docker-volume-vsphere/issues/1651)
+- Specifying "Datastore Cluster" name during volume creation is not supported. Datastore clusters (as a part of Storage DRS) is a VC feature and not available on individual ESX. [#556](https://github.com/vmware/docker-volume-vsphere/issues/556)
+
+**vsphere HA**
+
+- In vsphere HA cluster, volume remains in attached state if the esx on which vm is running goes down.  This happens only if  container is running with volume mounted and the ESX of the vm goes down or ESX is restarted. [#1675](https://github.com/vmware/docker-volume-vsphere/issues/1675) 
+
+**VM Reset**
+
+- Volume remains in attached state if 'Reset' of a vm is done while container is running. [#1638](https://github.com/vmware/docker-volume-vsphere/issues/1638)
+
+**Snapshot**
+
+- If vm is reverted to a snapshot while container is running, volume remains in attached state after revert has finished and volume removal will fail. [#1649](https://github.com/vmware/docker-volume-vsphere/issues/1649)
+
```

Comment:
Please mention that snapshots aren't supported for VMs that are running docker containers. Reasons being that if a VM is running a container and is snapped, then the snapped VMX configuration has the disk recorded in it and ESX will disallow detaching the vDVS volume when the container exits inside the VM and the plugin attempts to detach the volume. For this reason, snapshots of VMs taken after starting one or more containers isn't supported.